VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version       | iOS version | Android version | Common version | Play Billing Library version | Unity IAP compatibility       |
 | ------------- | ----------- | --------------- | -------------- | ---------------------------- | ----------------------------- |
+| 7.0.0 | [5.2.2](https://github.com/RevenueCat/purchases-ios/releases/tag/5.2.2) | [8.3.1](https://github.com/RevenueCat/purchases-android/releases/tag/8.3.1) | [13.0.0](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/13.0.0) | [7.0.0](https://developer.android.com/google/play/billing/release-notes) | Not compatible |
```

Comment:
Note I marked this as not compatible with Unity IAP since we support BC7 and Unity IAP only supports BC6